python/cudf/cudf/core/udf/utils.py

```diff
@@ -306,6 +253,16 @@ def _compile_or_get(
     we then obtain the return type from that separate compilation and
     use it to allocate an output column of the right dtype.
     """
+    # runtime check for CEC mode which is disabled for CUDA 12 for now
+    if cuda.cudadrv.driver.get_version() == (
+        12,
+        0,
+    ) and cuda.cudadrv.runtime.get_version() > (12, 0):
+        raise ValueError(
+            "Minor version compatibility not yet supported for "
+            "CUDA driver versions newer than 12.0"
+        )
```

Comment:
I believe this block resolves https://github.com/rapidsai/cudf/issues/13339, right? If so, can you update the PR description accordingly?

Reply:
Done!